docs/content/reference/ransomware.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Ransomware"
+date: 2021-06-23T18:13:59+05:30
+draft: true
+pre: '<i class="fas fa-lock"></i> '
+weight: 10
+---
+
+The Infection Monkey has the capability of simulating a ransomware attack on your network.
+All actions performed by the encryption routine are safe for production environments.
+
+To ensure minimum intereference and easy recoverability, the ransomware simulation will only run if
+it is configured properly. To do so, you must specify the path to a directory in the configuration.
+If no directory is specified, the simulation will not run.
```

Comment:
The simulation is more than just encryption, it is also propagation. It potentially includes changing wallpaper or enumerating files.
```suggestion
To ensure minimum intereference and easy recoverability, the ransomware simulation will encrypt
files if it the user specifies a directory that contains files that are safe to encrypt.
If no directory is specified, no files will be encrypted.
```

Reply:
I think that we also need to add that the user can't choose certain directories on his system.

Reply:
ATM, we're not planning on adding any such restriction.